com.github.sdbg.debug.ui/src/com/github/sdbg/debug/DebugLaunch.java

```diff
@@ -0,0 +1,37 @@
+package com.github.sdbg.debug;
+
+import com.github.sdbg.debug.ui.internal.DartUtil;
+import com.github.sdbg.debug.ui.internal.chrome.ChromeLaunchShortcut;
+import com.google.gdt.eclipse.core.sdbg.IDebugLaunch;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+
+public class DebugLaunch implements IDebugLaunch {
```

Comment:
The extension point will remain a single one. There is nothing stopping SDBG (or any other plugin) from contributing ***multiple*** implementations of IDebugLaunch - one for each supported browser.

If by interface assessors you mean more methods on the IDebugLaunch interface - each launching a specific browser - I think that would be an API leak. Why would GWT-Eclipse-Plugin would like to know ***what*** is being launched?